CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -25,6 +27,12 @@ def layer_test() -> str:
     return "hello world"
 
 
+@dataclass
+class Field_Validator:
+    is_valid: str
+    message: str
+
+
```

Comment:
nit: isn't the pre-commit hook removing extraneous white lines?

Reply:
I think 2 lines after classes are fine

All of the classes have 2 lines between them